pipeline/src/node_properties/track_properties.rs

```diff
@@ -0,0 +1,9 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+#[serde(tag = "tracker", rename_all = "snake_case")]
+pub enum TrackProperties {
```

Comment:
it'd be more future safe if we use a struct and have `tracker` as a field. Unless we're very sure this is the only data that will be needed here?

Reply:
likely will have more data in the future